cmd/agent/api/grpc.go

```diff
@@ -1,7 +1,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
+// Copyright 2016-2020 Datadog, Inc.
```

Comment:
This reverts #7332, we should not do it
```suggestion
// Copyright 2016-present Datadog, Inc.
```